java/connector-node/risingwave-sink-deltalake/src/main/java/com/risingwave/connector/DeltaLakeSink.java

```diff
@@ -80,7 +82,19 @@ public void write(Iterator<SinkRow> rows) {
                 case INSERT:
                     GenericRecord record = new GenericData.Record(this.sinkSchema);
                     for (int i = 0; i < this.sinkSchema.getFields().size(); i++) {
-                        record.put(i, row.get(i));
+                        Object values;
```

Comment:
Can we just check `row.get(i) instance of Timestamp` and call `getTime` directly rather than checking the data type with `convertType`?